.travis.yml

```diff
@@ -14,6 +14,10 @@ env:
 language: python
 python:
   - "2.7"
+matrix:
+  include:
+    - python: "3.4"
+      env: TEST_SUITE=mypyer
 # command to run tests
 script:
   - ./tools/travis/$TEST_SUITE
```

Comment:
Wouldn't it be confusing for people to talk about two things with the same name? People will always have to refer to the script as tools/travis/mypy instead of just mypyer. It's not a very big issue but some people might find it inconvenient in the future.

Are you sure I should change the name to tools/travis/mypy?